### DISCUSSION TITLE
NuGet Package 🤗

### DISCUSSION THREAD

**Discussion by ghost** [nigels-com/glew#299] [Ideas]
Recently I've been using VS and then I discovered the NuGet packages, super util when deploying to repositories or remote build services. 
So, I thought that could be awesome if you add a NuGet package from your win32 binaries, those are made to copy and run purposes. If you are interested, I already have done one to your last release and would be awesome if you are who distribute it.
[here is the link](https://www.nuget.org/packages/glew-2.2.0/)

**Comment by nigels-com** [comment]
Hello @nerdtronik thanks for the note about NuGet packages.
I'll think about how help publicise the availability of these, along with other channels that I know of.

This repository does not currently have a working Windows build using Appveyor, or similar.
It's probably solvable, the main issue tends to be the UNIX-style make, shell and perl scripted code generation.
(Contributions welcome, as always!)

I think it's would be worth mentioning this over at https://github.com/Perlmint/glew-cmake that is tracking the generated sources and can probably be built directly from the bundled Visual Studio projects, or in combination with cmake.

**Reply by ghost** [reply]
perfect, I'll talk to them. Thanks for your time and your amazing job.
